hashes/src/hmac/mod.rs

```diff
@@ -132,8 +132,10 @@ impl<T: GeneralHash> Hash for Hmac<T> {
 
     fn from_byte_array(bytes: T::Bytes) -> Self { Hmac(T::from_byte_array(bytes)) }
 
-    #[allow(deprecated)]
-    fn from_slice(sl: &[u8]) -> Result<Hmac<T>, FromSliceError> { T::from_slice(sl).map(Hmac) }
+    #[allow(deprecated)] // Because of `FromSliceError`.
```

Comment:
The method itself should be deprecated too though.

Reply:
I don't believe  so, its a trait method implementation so its deprecated already, right?